openapi/oi.json

```
@@ -646,74 +675,40 @@
                     "type": "array",
                     "items": {
                       "type": "string"
                     },
-                    "example": ["user1@example.org", "user2@example.org"]
-                  },
-                  "client_name": {
-                    "type": "string",
-                    "example": "https://example.org"
-                  },
-                  "subject_type": {
-                    "type": "string",
-                    "example": "pairwise"
-                  },
-                  "id_token_signed_response_alg": {
-                    "type": "string",
-                    "example": "RS256"
-                  },
-                  "token_endpoint_auth_method": {
-                    "type": "string",
-                    "example": "client_secret_basic"
-                  },
-                  "scope": {
-                    "type": "string",
-                    "example": "openid email profile openid_client_registration"
-                  },
-                  "organization": {
-                    "type": "string",
-                    "example": "https://organization.com"
-                  },
-                  "description": {
-                    "type": "string",
-                    "example": "Registered via OpenID Connect Dynamic Registration"
+                    "description": "Custom parameter which return all SPID attributes selected, https://docs.italia.it/italia/spid/spid-regole-tecniche/it/stabile/attributi.html",
+                    "example": [
+                      "spidCode",
+                      "name",
+                      "familyName",
+                      "fiscalNumber"
+                    ]
                   },
-                  "environment": {
+                  "logo_uri": {
                     "type": "string",
-                    "example": "ALL"
-                  },
-                  "master": {
-                    "type": "boolean",
-                    "example": false
+                    "description": "The RP logo URI",
+                    "example": "https://example.org/logo.png",
+                    "format": "uri"
                   },
-                  "spid_requested_attributes": {
+                  "default_acr_values": {
                     "type": "array",
                     "items": {
                       "type": "string"
                     },
-                    "description": "Custom parameter which return all SPID attributes selected, https://docs.italia.it/italia/spid/spid-regole-tecniche/it/stabile/attributi.html",
-                    "example": ["spidCode", "name", "familyName", "fiscalNumber"]
+                    "description": "list of the Authentication Context Class References that this RP supports",
+                    "example": [
+                      "https://www.spid.gov.it/SpidL1",
+                      "https://www.spid.gov.it/SpidL2",
+                      "https://www.spid.gov.it/SpidL3"
```

```suggestion
                      "L1",
                      "L2",
                      "L3"
```
For now we expose these values via an Enum. Let us reason together whether we prefer the whole string over just the levels.